pkg/detectors/auth0managementapitoken/auth0managementapitoken_integration_test.go

```diff
@@ -52,6 +52,7 @@ func TestAuth0ManagementApiToken_FromChunk(t *testing.T) {
 			want: []detectors.Result{
 				{
 					DetectorType: detectorspb.DetectorType_Auth0ManagementApiToken,
+					RawV2:        []byte(managementApiToken + domain),
```

Comment:
question: This might be a dumb question, but if the test fails, do we risk leaking the actual credential in the test output during the comparison?

Reply:
That's the approach for all the detectors integration tests 😅. We pull our secrets from GCP and use them for the testing.
But as you mentioned now I realised that's we may need to do something about it I guess.

Reply:
Maybe it's okay, given you need to have permissions to even access the secret from GSM.